kudu/README.md

```diff
@@ -42,6 +42,13 @@ Additional configurations:
 * `kudu_block_size`: The data block size used to configure columns. The default
   is 4096 bytes.
 
+To connect to Kudu on a cluster that requires Kerberos authentication, some additional 
+configuration is needed:
```

Comment:
Isn't this only necessary if you plan to run YCSB for longer than the expiration time of your Kerberos tickets (often 7 days)? Normally you should be able to just 'kinit' so that you have Kerberos credentails in your ticket cache, and then the client will pick them up with no other changes necessary.